py/torch_tensorrt/dynamo/utils.py

```diff
@@ -270,3 +271,25 @@ def function_wrapper(*args: Any, **kwargs: Any) -> Any:
         return function_wrapper
 
     return nested_decorator
+
+
+@torch.utils._python_dispatch._disable_current_modes()  # type: ignore
+def constant_fold(gm: torch.fx.GraphModule) -> Any:
+    cf = ConstantFolder(gm, skip_constructors=False)
+    cf.run()
+
+    for node, constant in cf.node_replacements.items():
+        replace_node_with_constant(gm, node, constant)
+
+    erased_params = []
+    for node in gm.graph.nodes:
+        if node.op == "get_attr" and len(node.users) == 0:
+            delattr(gm, node.target)
+            erased_params.append(node)
+
+    for node in erased_params:
+        gm.graph.erase_node(node)
+
+    gm.graph.eliminate_dead_code()
+    gm.graph.lint()
+    gm.recompile()
```

Comment:
Is now a utility in the lowering passes, so this can be removed

Reply:
done. rebased